src/constants/abi.js

```diff
@@ -0,0 +1,382 @@
+const abi = [
```

Comment:
please add as another line into https://github.com/ethfinex/community-gateway/blob/master/src/constants/abis.json instead of a separate file, so that its clear which contract the abi is for